docs/content/docs/build/auth.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Authentication & authorization"
+menuPosition: 3
+editor: tylerbutler
+---
+
+Security is critical to modern web applications. Fluid Framework, as a part of your web application architecture, is an
+important piece of infrastructure to secure. Fluid Framework is a layered architecture, and auth-related concepts are
+primarily a concern of the *driver* layer (see [Architecture]({{< relref "architecture.md" >}})). This means that the
+specifics of authentication could differ based on the Fluid service and its driver.
+
+The information below is based on Azure Fluid Relay service but also applies to
+[Tinylicious]({{< relref "tinylicious.md" >}}). Other Fluid services may differ.
+
+## Azure Fluid Relay service
+
+{{< include file="_includes/frs-onboarding.html" safeHTML=true >}}
+
+Each FRS tenant you create is assigned a *tenant ID* and *secret key*.
+
+The secret key is a *shared secret*. Your app/service knows it, and FRS knows it. This means that you can sign data
```

Comment:
```suggestion
The secret key is a *shared secret*. Your app/service knows it, FRS knows it, and since it is uniquely tied to your tenant, its usage guarantees that any request signed by it are coming from an authorized user of this tenant . This means that you can sign data
```

Reply:
+1 on Sumedh's edit. We should make it clear that these secrets are tenant specific.